tooling/debugger/src/lib.rs

```diff
@@ -85,7 +87,12 @@ impl<'backend, B: BlackBoxFunctionSolver> DebugContext<'backend, B> {
                 let start = loc.span.start() as usize;
                 let end = loc.span.end() as usize;
                 println!("At {}.nr:{start}-{end}", file.path.as_path().display());
-                println!("\n{}\n", &source[start..end]);
+                println!(
+                    "\n{}{}{}\n",
+                    &source[0..start],
+                    &source[start..end].to_string().reversed(),
+                    &source[end..]
```

Comment:
```suggestion
                    &source[0..start].to_string().dimmed(),
                    &source[start..end],
                    &source[end..].to_string().dimmed()
```

This is personal taste but I quite like this display as it's easier on the eyes while drawing more attention to the line in question.

example: ![image](https://github.com/noir-lang/noir/assets/15848336/a5fbb8b6-45a3-487f-a3de-1807e15abe3b)

Reply:
It actually does look much better, I'll apply your suggestion!